presto-main/src/main/java/com/facebook/presto/operator/WindowOperator.java

```diff
@@ -193,6 +193,7 @@ public OperatorFactory duplicate()
     private final PageBuilder pageBuilder;
 
     private final DriverWindowInfoBuilder windowInfo;
+    private DriverWindowInfo driverWindowInfo;
```

Comment:
Change WindowInfo to materialize on operator close

This minimizes unnecessary data transfer and computation.